code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -277,6 +294,14 @@
 	base_icon_state = "snail"
 	list_reagents = list(/datum/reagent/snail = 10)
 
+/obj/item/reagent_containers/hypospray/medipen/snail/examine()		// no LCD screen for the snails :(
```

Comment:
This will have the LCD screen because you're calling parent

Reply:
why did you mark my review as resolved when it isnt

![image](https://user-images.githubusercontent.com/53777086/187841846-55f01020-d29b-4d2f-ace6-b6063b54683d.png)